test/descriptive.test.js

```diff
@@ -44,4 +45,15 @@ describe('Descriptive Statistics', () => {
     testUndefinedWithNullable(product);
   });
 
+  test('Summation', () => {
+    expect(sum([0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0, 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7])).toBe(15.3);
```

Comment:
Please test at least two valid inputs. Also, I tend to test the case I add in the documentation. Could you please do the same ?